Assets/Plugins/iOS/Shopify/errorPage.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" /> 
+<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,user-scalable=no" />
+<link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/1.0.2/polaris.css" />
```

Comment:
I wonder if instead of injecting the `CSS` we can link to the `CSS` using the bundle path